run_cdc_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+export FLASK_ENV=test
+
+python3 -m venv .env
+source .env/bin/activate
+
+python3 -m pytest server/webdriver/cdc_tests/
```

Comment:
Nit: move to `./run_test.sh` with a new `--custom` flag

Reply:
Good callout- i think moving it to the standalone folder sounds good!

Reply:
Done! While I was in the standalone dir I saw more extensive sanity tests for main DC (I think these are the ones Bo mentioned on the other thread). As a follow-up I may try to reuse those for custom DC.